config.toml

```diff
@@ -45,17 +45,24 @@ time_format_blog = "Monday, January 02, 2006"
 description = "Production-Grade Container Orchestration"
 showedit = true
 
-latest = "v1.10"
+latest = "v1.11"
 
-fullversion = "v1.10.3"
-version = "v1.10"
+fullversion = "v1.11.0"
+version = "v1.11"
 githubbranch = "master"
-docsbranch = "master"
+docsbranch = "release-1.11"
 deprecated = false
 currentUrl = "https://kubernetes.io/docs/home/"
 nextUrl = "http://kubernetes-io-vnext-staging.netlify.com/"
 githubWebsiteRepo = "github.com/kubernetes/website"
 
+[[params.versions]]
```

Comment:
Since we are only supporting current + last 4, remove the section for v1.7 starting on line 87.

Reply:
@chenopis Did you mean removing the section for v1.6?

Reply:
Yup, sorry. My counting fingers aren't working today. :( I meant for v1.6 starting on line 94.